feathr_project/feathr/spark_provider/_synapse_submission.py

```diff
@@ -178,7 +178,9 @@ def wait_for_completion(self, timeout_seconds: Optional[float]) -> bool:
                 return True
             elif status in {LivyStates.ERROR.value, LivyStates.DEAD.value, LivyStates.KILLED.value}:
                 logger.error("Feathr job has failed.")
-                logger.error(self._api.get_driver_log(self.current_job_info.id).decode('utf-8'))
+                error_msg = self._api.get_driver_log(self.current_job_info.id).decode('utf-8')
+                logger.error(error_msg)
+                logger.error("The size of the whole error log is: {}. It can be larger than size limit of log. If you cannot see the whole log, you may either extend setting for size limit, or follow the link above to check it.", len(error_msg))
```

Comment:
logger.error("The size of the whole error log is: {}. The logs might be truncated in some cases (such as in Visual Studio Code) so only the top a few lines of the error message is displayed. If you cannot see the whole log, you may want to extend the setting for output size limit.", len(error_msg))

Reply:
Changed.